sycl/doc/extensions/SPIRV/SPV_INTEL_vector_compute.asciidoc

```diff
@@ -91,6 +91,11 @@ Specifies the size of shared memory required by kernel, in bytes.
 
 See the client API specification for more detail.
 | *VectorComputeINTEL* | <<Literal, 'Literal'>> _Size_
+| 8088 | *VectorComputeFastCompositeKernelINTEL* +
```

Comment:
Was this number and numbers below reserved?

Reply:
It's a typo, thanks for noticing. The number is fixed in the latest commit.

Reply:
Thanks. I assume, since it's a fresh spec change, we don't have yet generated SPIR-V modules in user's code and we can change it in the translator, right? I'll update it in the translator in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/810